### PR TITLE
T_ch2_c10s

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
- [x] 1 NEED DISCUSSION: https://github.com/GNAQ/CN_missemi/blob/T_ch2_c10s/ch2/%5B478-538%5DCHN.txt#L59
- [x] Review
- [x] 统一翻译 ~~，大小写~~

### CODE REVIEW

File: ch2/[478-558]CHN.txt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+如果我最后一次执行 ':q' 
+
+好的，现在 Vim 退出了
+
+如果您不想太多次按 ':q' ...
+
+【510】那么接下来，这里有三个独立的 window
+
+如果我执行 ':qa' ，则退出所有window
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:52]:
```suggestion
如果我执行 ':qa' ，则退出所有 window
```

File: ch2/[478-558]CHN.txt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+而不是箭头键来操作。
+
+尽管默认情况下它们确实起作用，但请尽量避免使用它们
+
+因为您不需要将手一直移到箭头键的位置
+
+这浪费了大量时间，对吗？
+
+ hjkl 就在你手指的那一行。
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:124]:
```suggestion
hjkl 就在你手指的那一行。
```

**Review by ksyx — COMMENTED** [review]
fix spaces

**Review by GNAQ — APPROVED** [review]
step to Kernel_bin to resolve REVIEW tag

File: ch2/[478-558]CHN.txt [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+这个命令将关闭所有打开的 window
+
+【513】好，现在回答你的问题
+
+Normal 模式实际上是做什么的
+
+这是 Vim 中另一个非常酷的想法
+
+我认为这实际上是该程序最有趣的想法
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:68]:
```suggestion
我认为这实际上是该程序最有趣的设计
```

File: ch2/[478-558]CHN.txt [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+【518】就像，诸位都是程序员，诸位喜欢编程一样
+
+Vim 的设计理念是 Vim 的 Normal 模式
+
+例如 Vim 的接口，本身是一种编程语言
+
+【522】让我重复一遍
+
+这就是 Vim 有趣的核心哲学：它的接口是种编程语言 [*]
+# REVIEW - 要不要解释「它的接口」？
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:79]: 交互界面 
交互方式

File: ch2/[478-558]CHN.txt [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+这意味着不同的按键组合具有不同的效果
+
+并且一旦您掌握了这些组合的、不同的效果
+
+就可以把它们拼到一起
+
+就像在编程语言中一样
+
+您可以学习不同的功能和内容
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:91]: ~~太您了~~

这样语气有点怪

File: ch2/[478-558]CHN.txt [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+【544】因此，让我们深入研究 Normal 模式怎么运作的
+
+您可以尝试跟着一起做
+
+例​​如，在 Vim 中随便打开一些文件
+
+并跟着我键入一些组合键
+
+【549】可能您最想做的一件事就是
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:121]:
```suggestion
【549】可能一上来你最想做的事就是
```

File: ch2/[478-558]CHN.txt [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+在 buffer 中移动光标
+
+比如上下左右地移动光标
```
Comment (ksyx) [inline ch2/[478-558]CHN.txt:125]:
```suggestion
在 buffer 中上下左右地移动光标
```

File: ch2/[478-558]CHN.txt (hunk 3 above)
Comment (GNAQ) [reply ch2/[478-558]CHN.txt:68]: OK, nice

File: ch2/[478-558]CHN.txt (hunk 5 above)
Comment (GNAQ) [inline ch2/[478-558]CHN.txt:91]:
```suggestion
并且一旦你掌握了这些组合的、不同的效果

就可以把它们拼到一起

就像在编程语言中一样

你可以学习不同的功能和内容
```

File: ch2/[478-558]CHN.txt (hunk 7 above)
Comment (GNAQ) [reply ch2/[478-558]CHN.txt:125]: 忠于原文

File: ch2/[478-558]CHN.txt (hunk 4 above)
Comment (GNAQ) [reply ch2/[478-558]CHN.txt:79]: @CoelacanthusHex @kernelbin

**Review by GNAQ — COMMENTED** [review]
UNSOLVED [是否解释接口](https://github.com/CN-missemi/CN_missemi/pull/4/files/23871e678045f56ce212d6cd608f1c4f9dfc848d#r580073612)

**Review by GNAQ — COMMENTED** [review]
UNSOLVED [是否解释接口](https://github.com/CN-missemi/CN_missemi/pull/4/files/23871e678045f56ce212d6cd608f1c4f9dfc848d#r580073612)

**Review by kernelbin — APPROVED** [review]

File: ch2/[478-558]CHN.txt (hunk 4 above)
Comment (GNAQ) [reply ch2/[478-558]CHN.txt]: 依据维基修改了接口的注释

**Review by GNAQ — APPROVED** [review]
All fixed.

**Pull request merged**